.github/workflows/integration.yml

```diff
@@ -27,5 +27,5 @@ jobs:
       - name: Test
         env:
           MNEMONICS: ${{ secrets.MNEMONICS }}
-          NETWORK: ${{ secrets.NETWORK }}
+          NETWORK: dev
```

Comment:
please make sure there's an issue exist to change that back in the future

Reply:
- https://github.com/threefoldtech/grid3-go/issues/113